src/renderers/shaders/ShaderChunk/begin_vertex.glsl

```diff
@@ -1,2 +1,28 @@
 
+#ifndef INSTANCE_TRANSFORM
+
 vec3 transformed = vec3( position );
+
+#else
+
+#ifndef INSTANCE_TRANSFORM_DEFINED
```

Comment:
@mrdoob 

I'm going back to this, since now i think the matrix stuff happens twice...